frontend/packages/kubevirt-plugin/src/components/vm-details.tsx

```diff
@@ -0,0 +1,155 @@
+import * as React from 'react';
```

Comment:
Can we move this to `./vm/` folder? - to distinguish between vm and vm template files in the future

Reply:
:+1: `public/components/*` seem to use the plural for directories, so I'd suggest using `components/vms` and `components/vm-templates` for better organization.